templates/items/backpack.hbs

```diff
@@ -73,7 +83,7 @@
             </div>
 
             <div class="form-group stacked">
-                <label>{{localize 'DND5E.ItemContainerProperties'}}</label>
+                <label>{{localize 'DND5E.ItemContentProperties'}}</label>
```

Comment:
I think weightless contents should be merged into properties. We can add a migration to add it to the properties set if it is checked and then define a getter to return the old property.

Something like this can be defined in `prepareDerivedData` on `ContainerData`:

```javascript
Object.defineProperty(this.capacity, "weightless", {
  value: this.properties.has("weightlessContents"),
  writable: false
});
```

Reply:
I may be missing how we can both define the property yet be rid of it if it is `true`.

Reply:
We should be able to:
1. Remove `weightless` from the schema
2. Add a migration that checks to see if `source.capacity.weightless` is `true` and add that to `source.properties` if so, otherwise do nothing
3. Then use `defineProperty` in `prepareDerivedData` to create a fallback value (though you'd probably want to rewrite what I put above to use `get() { }` rather than `value:` so you can log a compatibility warning)

Reply:
One issue I am seeing, just doing steps 1 and 2, is that if `source.capacity.weightless === true`, then `source.properties === undefined`.

Reply:
After thinking on it some, I am agreeing it should be part of `properties`. It is entirely specific to containers and the contents that it holds.